Jenkinsfile

```diff
@@ -32,6 +32,7 @@ def presubmit(gitUtils, bazel, utils) {
       sh('bin/linters.sh')
     }
     stage('Bazel Test') {
+      sh('source istio.VERSION')
```

Comment:
Let s create a script for end to end testing in another PR. Just tag the sample test as manual in the BUILD file.